controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -191,7 +191,7 @@ module.exports = async (req, res) => {
     //END LOCATIONS
 
     //OTHER USERS
-    if (page == 1 && topics) {
+    if (topics) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `page == 1` condition means that the admin ID will be added to the `allUserIds` array regardless of the page number. This could potentially alter the pagination behavior. If you want to maintain the previous behavior, consider reintroducing this condition.
```diff
- if (page == 1 && topics) {
+ if (topics) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->